### DISCUSSION TITLE
My page isn't being published

### DISCUSSION THREAD

**Discussion by OliverFoxPackham** [community/community#23891] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve posted a page before so I did the same thing and it didn’t work, I double checked the documentation and it seemed fine. I read that someone had the same problem but it was fixed when they chose a jekyll theme but that didn’t work either. my page says it’s published but I get a 404 message when I click the link. I also opened it in incognito to make sure it wasn’t just the cashed version. <a href="https://github.com/OliverFoxPackham/mysced/settings/pages" rel="noopener nofollow ugc">https://github.com/OliverFoxPackham/mysced/settings/pages</a></p>
<p>thank you for the help.<br>
-Oliver</p>

**Comment by jfullstackdev** [comment]
<p>hello there, others cannot see that settings of your pages, but I just checked your site and it’s really 404,</p>
<p>so please provide further details and screenshots so that we can help</p>

**Comment by OliverFoxPackham** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/b/3b9b96e7f027470a508e845310b143901752ac91.png" data-download-href="/uploads/short-url/8vjwdCyyNktqaInau2JRSfuk7xn.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/69836083/181079864-d721155c-5cfc-4ebb-bb69-89d7b3c4e77f.png" alt="image" data-base62-sha1="8vjwdCyyNktqaInau2JRSfuk7xn" width="629" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3b9b96e7f027470a508e845310b143901752ac91_2_629x499.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3b9b96e7f027470a508e845310b143901752ac91_2_943x748.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/b/3b9b96e7f027470a508e845310b143901752ac91.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3b9b96e7f027470a508e845310b143901752ac91_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1179×937 55.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/f/7fa3be2e45c4cf335f3eb476446709ca81db97d4.png" data-download-href="/uploads/short-url/id9shVWWslsvvq96OfHjanUnqao.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181079865-151b4439-e9ce-4a8a-af44-1cd2f32ccba3.png" alt="image" data-base62-sha1="id9shVWWslsvvq96OfHjanUnqao" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/f/7fa3be2e45c4cf335f3eb476446709ca81db97d4_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/f/7fa3be2e45c4cf335f3eb476446709ca81db97d4_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/f/7fa3be2e45c4cf335f3eb476446709ca81db97d4_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/f/7fa3be2e45c4cf335f3eb476446709ca81db97d4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 183 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>your site is published, you just did not get the correct URL,</p>
<p><a href="https://oliverfoxpackham.github.io/mysced/html/index.html" rel="noopener nofollow ugc">https://oliverfoxpackham.github.io/mysced/html/index.html</a></p>
<p>if you want to display it on the root <code>mysced</code>, place the <code>index.html</code> at the root of your repo and change all the paths for images and links,</p>

**Comment by OliverFoxPackham** [comment]
<p>Thank you for the help, I really apricate it.</p>
